gapic-generator-java/src/test/java/com/google/api/generator/gapic/composer/grpcrest/GrpcRestTestProtoLoader.java

```diff
@@ -58,6 +60,13 @@ public GapicContext parseShowcaseEcho() {
     ServiceDescriptor echoServiceDescriptor = echoFileDescriptor.getServices().get(0);
     assertEquals("Echo", echoServiceDescriptor.getName());
 
+    String serviceYamlFileName = "showcase_v1beta1.yaml";
```

Comment:
I probably missed this in previous reviews, instead of `showcase_v1beta1.yaml`, can we rename it to reflect the fact that this yaml is only used by Echo at this point?

Reply:
Sure, can rename to `echo_v1beta.yaml`. We can rename it back to `showcase_v1beta1.yaml` if other tests need to use this file again for other showcase protos.